Gemfile

```diff
@@ -1,18 +1,21 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.6.6'
+ruby '2.7.4'
+
+# stdlib gem to silence warnings
+gem "net-http"
```

Comment:
Webpacker uses `net/http` without depending on `net-http`, which causes conflicts & warnings when other gems _do_.

See https://github.com/ruby/net-imap/issues/16 for additional discussion.